recipes-core/systemd/systemd_%.bbappend

```diff
@@ -12,6 +12,12 @@ do_install_append() {
     	echo "Address=${STATIC_IP_ADDRESS}/24" >> ${D}${sysconfdir}/systemd/network/10-pelux-ethernet.network
     fi
     ln -s /run/systemd/resolve/resolv.conf ${D}${sysconfdir}/resolv.conf
+
+    # Create mutiple consoles
+    tmp="${TTY_LIST}"
+    for ttydev in $tmp ; do
+        ln -sf ${systemd_unitdir}/system/getty@.service ${D}${sysconfdir}/systemd/system/getty.target.wants/getty@$ttydev.service
```

Comment:
You have probably tried this, but just want to check if this is the preferred place to put those service files, as opposed to, for example, `multi-user.target.wants`.

Reply:
These service files will get created as part of getty.target. So whichever run level includes getty.target, These service files will be created.

Reply:
@GordanM  Yes still now my changes on creating symlink do not reflect in subsequent build.

Reply:
I would try to do a ``rm -rf ${D}${sysconfdir}/systemd/system/getty.target.wants/`` before the symlink step then.

Reply:
@GordanM  @tobsan , The image is containing my changes after making change to creating symlink. It is just that the dd command that I use to write the image to pendrive is not writing the newly introduced symlinks. If I am formatting the pendrive and then using dd cmd to write the image, it is actually showing the newly introduced symlinks as well.

Reply:
That's strange. Can you try to add `FILES_${PN} += ${sysconfdir}/systemd/system/getty.target.wants` and see if that changes anything?